doc/getting-started.md

```diff
@@ -82,24 +93,65 @@ most simple capabilities is to add a delay.
 
 ```
 # First, create a rule file
-cat <<EOF > /tmp/hello-5s-rule.yaml
+cat <<EOF > /tmp/couch-5s-rule.yaml
 type: route-rule
-name: hello-5s-rule
+name: couch-5s-rule
 spec:
-  destination: hello.default.svc.cluster.local
+  destination: couchdb.default.svc.cluster.local
   http_fault:
     delay:
       percent: 100
       fixed_delay_seconds: 5
 EOF
 
 # Give the file to istioctl
-istioctl create -f /tmp/hello-5s-rule.yaml
+istioctl create -f /tmp/couch-5s-rule.yaml
 ```
 
-Wait a few seconds, then issue `curl $IP:$NODEPORT` again.  This time it takes five seconds for *frontend* to reach *hello*.
+Wait a few seconds, then issue `curl $IP:$NGINX_NODEPORT` again.  This time it takes five seconds for *frontend* to reach *hello*.
 
+```
+time curl -i $IP:$NGINX_NODEPORT
+```
+
+The rule file defines a 5 second delay to be added to 100% of network traffic with the destination host name 
+"couchdb.default.svc.cluster.local".  Why the long destination?  Although the NGINX configuration only uses "couchdb", the Istio configuration
+requires the full format "&lt;host&gt;.&lt;namespace&gt;.svc.cluster.local" when running on Kubernetes.  The _kube-dns_ service on
+Kubernetes sets up "<namespace>.svc.cluster.local" as the DNS search path so the names are equivelent.
+
+## Cleanup
+
+To remove the rules, deployments and services used in this tutorial:
+
+```
+istioctl delete route-rule couch-5s-rule
+kubectl delete service couchdb
+kubectl delete deployment couchdb
+kubectl delete service nginx
+kubectl delete deployment nginx
+kubectl delete -f kubernetes/istio-install # OPTIONAL
+```
 
 ## Discussion
 
-TODO
+We have seen two web services, CouchDB and NGINX, connected via an Istio proxy mesh.  We have
+used the proxy mesh to introduce a delay in all communications to CouchDB.  Normally Istio would
+not be used to introduce delays &mdash; we use that feature for testing the behavior of microservice
+applications.
+
+A later tutorial will show how to use the features of Istio for administering cloud systems.
+We will demonstrate rolling updates with zero down time.  We will also show the introduction of
+feature flags on unmodified applications.
```

Comment:
I would not call them feature flags. It will throw off anyone who has used LaunchDarkly or any standard feature flag SDK. Simply calling it as routing based on request headers is enough to convey the idea, IMO